[BE] Validate a job

nspc97 opened the pull request:
[BE] Validate a job
https://trello.com/c/du6VOqEr/101-be-validate-a-job

Commit:
[BE] Validate a job
https://trello.com/c/du6VOqEr/101-be-validate-a-job

MihaiDruta commented on src/Controller/ActivityController.php:
```diff
@@ -1536,4 +1536,103 @@ public function getActivitiesForValidation(ActivityRepository $activityRepositor
     }
+
+    /**
+     * Validate an applicant.
```
Should be: validate a Job

Commit:
[BE] Validate a job
https://trello.com/c/du6VOqEr/101-be-validate-a-job

SpartakusMd reviewed (changes requested):
Add tests for validation:
- Successful test
- one test per each failing case

Commit:
[BE] Validate a job
https://trello.com/c/du6VOqEr/101-be-validate-a-job

SpartakusMd commented on src/DataFixtures/UserFixtures.php:
```diff
@@ -22,13 +25,25 @@ class UserFixtures extends Fixture
         $this->encoder = $encoder;
+        $users = $userRepository->findAll();
+        foreach ($users as $user) {
+            $userRepository->unassignProjectManager($user);
```
Why are you doing this in fixtures? Everything from database is removed at each load of fixtures.

SpartakusMd commented:
Travis build is failing. Check why and fix the issue.

Commit:
[BE] Validate a job
https://trello.com/c/du6VOqEr/101-be-validate-a-job

- Added a new command app:fixtures:load
